cli/generators/crud/templates/appService.js

```diff
@@ -0,0 +1,81 @@
+import fetch from 'isomorphic-fetch';
+import * as endpoints from './apiEndpoints';
+
+class <%= ucName %>Service {
+  static load<%= pluralizedUcName %>() {
+    const request = new Request(`${endpoints.BASE_URL}${endpoints.GET_<%= pluralizedName.toUpperCase() %>}`, {
+      method: 'GET'
+    });
+
+    return fetch(request).then(response => {
+      return response.json();
+    }).catch(error => {
+      return error;
+    });
```

Comment:
Well, yes, that could be simplified but I have mixed feelings on doing something like `return fetch(request);`. I think the Service should provide the data already "processed" or "digested".

I think the catch could be removed for now and we can use it on the next layer or doing something else here when we need.

What do you think folks?